packages/varlet-cli/site/pc/components/AnimationBox.vue

```diff
@@ -1,23 +1,31 @@
 <script lang="ts">
 import { defineComponent, onMounted, onUnmounted, ref, useAttrs } from 'vue'
-import { animationBoxData, animationEl, animationElClientRect, isMountedCount } from '../floating'
+import { animationBoxData, animationEl, isMountedCount } from '../floating'
 
 export default defineComponent({
   name: 'AnimationBox',
+  props: {
+    /**
+     * If this component is in a `position: fixed` container,
+     * this value must be to set `true`
+     */
+    fixed: {
+      type: Boolean,
+      default: false
+    }
+  },
```

Comment:
It's shoud be a No parameter component

Reply:
I haven't thought of a good solution yet.

By route name?
After Vue Router handles the scrolling?

Reply:
This bug only needs to adjust the relative position of the logo component to the screen, and it doesn't need to pay attention to whether it floats or not, because I have already dealt with this problem.